src/type/definition.ts

```diff
@@ -570,13 +570,12 @@ export class GraphQLScalarType {
   }
 
   toConfig(): {
-    ...GraphQLScalarTypeConfig<unknown, unknown>,
     serialize: GraphQLScalarSerializer<unknown>,
     parseValue: GraphQLScalarValueParser<unknown>,
     parseLiteral: GraphQLScalarLiteralParser<unknown>,
     extensions: Maybe<ReadOnlyObjMap<unknown>>,
     extensionASTNodes: ReadonlyArray<ScalarTypeExtensionNode>,
-  } {
+  } & GraphQLScalarTypeConfig<unknown, unknown> {
```

Comment:
change to inheritance after 998bea680d6e11e1c055a400a887a9539de08f75 is added to `16.x.x`

Reply:
you should do that for `definition.ts` and `directives.ts`